toc/working-groups/concourse.md

```diff
@@ -0,0 +1,60 @@
+# Concourse: Working Group Charter
+
+## Mission
+
+Provides the Concourse CI/CD platform.
+
+## Goals
+* Provides a stable and up to date CI/CD platform.
+
+## Scope
+* Maintain public roadmaps for Concourse.
+* Provide the community with regular releases of Concourse.
+* Provide timely releases to address CVEs.
+* Provide the community with technical and end user documentation.
+* Collaborate with other working groups to continue to have first-class support with Cloud Foundry.
+
+## Non-Goals
+
+* Require other working groups or projects to use Concourse.
+
+## Working Group Governance Structure
+
+[Working group governance structure](https://github.com/concourse/governance)
+
+## Membership
+
+[Working group membership](https://github.com/concourse/governance/blob/master/teams/maintainers.yml)
+
+## Roles & Technical Assets
+
+Technical assets for the Concourse working group are contained in the following Github Organizations:
+
+* [github.com/concourse](https://github.com/concourse)
+
+```yaml
+name: Concourse
+execution_leads:
+- name: Derek Richard
+  github: drich10
+- name: Taylor Silva
+  github: taylorsilva
+technical_leads:
+- name: Derek Richard
+  github: drich10
+- name: Taylor Silva
+  github: taylorsilva
+areas:
+- name: Maintainers
```

Comment:
This probably should be something like `Core` so that in the future, there can be other area's that are not part of the `Core` like for a specific set of resources.

Reply:
Makes sense, updated